networkx/algorithms/bipartite/matching.py

```diff
@@ -378,7 +379,8 @@ def _alternating_dfs(u, along_matched=True):
     # check for alternating paths starting with edges not in the
     # matching.
     return _alternating_dfs(v, along_matched=True) or _alternating_dfs(
-        v, along_matched=False
+        v,
+        along_matched=False,
```

Comment:
This passes the linters in its original form...  Is there a reason to change it?

Reply:
My code didn't pass the black linter, so I reformatted it using the black linter and maybe it caused these changes...
So I guess there isn't any reason to change it.